### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Kirtikagoyal**
## Short Description
updated the documentation.

## Changes proposed
Updated the readme documentation of spaces lounge.
added banner of spaces lounge.

## Screenshots
image
image

### CODE REVIEW

**Comment by alisamadiii** [comment]
Can you send us some screenshots from it?

File: README.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
-![Figma](https://img.shields.io/badge/Figma-305FCB?style=for-the-badge&logo=figma&logoColor=white)
+## Tech stack
+Spaces Lounge is built using the following technologies:
 
 - Frontend - **Next.js**
 - CSS Framework - **Tailwind CSS**
 - Design & Prototype - **Figma**
 
-## Running the project
+##Running the project
```
Comment (alisamadiii) [inline README.md]: There has to be a space
```diff
- ##Runn...
+ ## Runn...
```

**Review by alisamadiii — CHANGES_REQUESTED** [review]
It is looking great, but need one change to be made.
You did a great job, Well done.

**Review by alisamadiii — APPROVED** [review]
LGTM 🚀

**Comment by alisamadiii** [comment]
Thanks for your contributions,
You did a great job.

**Comment by Kirtikagoyal** [comment]
Thankyou Ali

**Pull request merged**